### PR TITLE
activemq Run as nonroot - melindam repository

### PR DESCRIPTION
**Pull request by melindam**

### CODE REVIEW

**Comment by melindam** [comment]
Let me know the changes scottymarshall - and that I have added you successfully as a "Assignee"

File: templates/default/activemq.xml.erb [hunk 1]
```diff
@@ -128,17 +128,44 @@
         -->
         <transportConnectors>
             <!-- DOS protection, limit concurrent connections to 1000 and frame size to 100MB -->
-            <transportConnector name="openwire" uri="tcp://0.0.0.0:61616?maximumConnections=1000&amp;wireformat.maxFrameSize=104857600"/>
```
Comment (scottymarshall) [inline templates/default/activemq.xml.erb:131]: Put the if statments around amqp and ssl, too

File: templates/default/activemq.xml.erb [hunk 2]
```diff
@@ -128,17 +128,44 @@
...
         <shutdownHooks>
             <bean xmlns="http://www.springframework.org/schema/beans" class="org.apache.activemq.hooks.SpringContextHook" />
         </shutdownHooks>
+		
+		<% if node['activemq']['simple_auth'] -%>
+	    <plugins>
+		<!-- ADDING FROM SIMPLE AUTHENTICATION EXAMPLE -->
+		<simpleAuthenticationPlugin>
+	    	  <users>
+	            <authenticationUser username="system" password="manager" groups="users,admins"/>
```
Comment (scottymarshall) [inline templates/default/activemq.xml.erb]: Remove all the those other users.  In fact, you should make this in to a loop that grabs the user from either a databag or an attribute.  That would take the "broker" code out of the public arena.

**Comment by melindam** [comment]
closing for new fixes to be reviewed

**Pull request closed without merging**